lib/gds_api/json_client.rb

```diff
@@ -64,8 +64,8 @@ def patch_json(url, params, additional_headers = {})
       do_json_request(:patch, url, params, additional_headers)
     end
 
-    def delete_json(url, additional_headers = {})
-      do_json_request(:delete, url, nil, additional_headers)
+    def delete_json(url, params = {}, additional_headers = {})
+      do_json_request(:delete, url, params, additional_headers)
```

Comment:
Did you see the method below this one? 

I'm assuming the deprecation notice is superseded by a later RFC where it says it's ok about bodies in delete. Might need a bit of cleanup here.

Reply:
Ah, I feel so blind, but also naughty 😆 .

Reply:
Updated.